src/main/java/net/glowstone/entity/projectile/GlowEnderPearl.java

```diff
@@ -27,7 +31,7 @@
      * @param location the position and facing of the thrower
      */
     public GlowEnderPearl(Location location) {
-        this(location, 2.0f);
+        this(location, 3.0f);
```

Comment:
Where does the health value come from?

Reply:
I think you mean https://github.com/Glowminers/Glowstone/blob/b36962cb2d0476a9bfadcd83437460eea94ddaf8/src/main/java/net/glowstone/entity/projectile/GlowEnderPearl.java#L103-L105.
When ender pearl collide with an entity, it counts as a punch from the thrower to the entity. For example, it makes wolf angry at the thrower and decrease reputation on villagers. But the entity takes no actual damage.

Reply:
Ah no, I was reading the `3.0f` as the health of the EnderPearl, because other entities generally pass their maxHealth in the constructor via `super(location, health)`. My mistake